fighters/gamewatch/src/acmd/specials.rs

```diff
@@ -175,7 +175,10 @@ unsafe fn gamewatch_special_s7_game(fighter: &mut L2CAgentBase) {
 unsafe fn gamewatch_special_s8_game(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
+    frame(lua_state, 16.0);
     if is_excute(fighter) {
+        WorkModule::on_flag(boma, *FIGHTER_GAMEWATCH_STATUS_SPECIAL_S_FLAG_FIX_PANEL);
+        VisibilityModule::set_default_int64(boma, Hash40::new("panel").hash as i64);
```

Comment:
you can just do `hash40("panel") as i64` here

Reply:
Old script repo moment